sklearn/tests/test_docstring_parameters.py

```diff
@@ -277,6 +266,13 @@ def test_fit_docstring_attributes(name, Estimator):
     if Estimator.__name__ == "MDS":
         est.set_params(normalized_stress="auto")
 
+    # Low max iter to speed up tests
+    if "max_iter" in est.get_params():
+        est.set_params(max_iter=2)
+
+    if "random_state" in est.get_params():
+        est.set_params(random_state=0)
+
```

Comment:
this is not really necessary since nothing we check depends on the seed. We usually try to avoid randomness but here we don't really care. Should I keep this addition or remove it ?